### PR TITLE
Add in github actions, templates and travis

### PR DESCRIPTION
**Pull request by CoMPaTech**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+If applicable, add screenshots to help explain your problem.
+
+**Home Assistant Core (please complete the following information):**
+ - Version: [e.g. `0.108.2`]
+ - Deployment: [e.g. 'raspberry/hassio' or 'py-venv']
+ - Installed plugwise version: [e.g. 'plugwise-beta or HA-core']
+ - Installed through [e.g. 'HACS', 'Core' or manual custom_component]
+
+**Smile (please complete the following information):**
+ - Sort of Smile: [e.g. Adam/Anna/P1]
```
Comment (bouwew) [inline .github/ISSUE_TEMPLATE/bug_report.md:41]: Type of Plugwise device: [Adam/Anna/P1/Stick/Stretch)

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Home Assistant (please complete the following information):**
+ - Version: [e.g. `0.108.2`]
+ - Deployment: [e.g. 'raspberry/hassio' or 'py-venv']
+ - Installed plugwise version: [e.g. 'plugwise-beta or HA-core']
+ - Installed through [e.g. 'HACS', 'Core' or manual custom_component]
+
+**Smile (please complete the following information):**
+ - Sort of Smile: [e.g. Adam/Anna/P1]
```
Comment (bouwew) [inline .github/ISSUE_TEMPLATE/feature_request.md:26]: See previous comment.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (CoMPaTech) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Improved

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (CoMPaTech) [reply .github/ISSUE_TEMPLATE/feature_request.md]: And also copy/paste improved

**Pull request merged**